tests/backtest/test_trade.py

```diff
@@ -0,0 +1,23 @@
+"""Test the Trade object."""
+
+import datetime
+
+from coin_test.backtest import Trade
+
+symbol = "BTC"
+side = "buy"
+type_ = "market"
+price = 10
+timestamp = datetime.datetime.fromtimestamp(int("riddle", 36))
```

Comment:
If you foresee more tests using these variables then consider also making these fixtures. Otherwise, might be alright as locals vars.

Reply:
Looking at this more, these might be best as local variables - I think it's alright to have some duplication in this regard. Took a look at [these tests](https://github.com/centerofci/mathesar/blob/d19bed1350e6f0d42e2632bc1a36179295db0c0d/mathesar/tests/api/test_column_api.py#L261), for reference, which do generally just define things locally.

Reply:
Done